confgenerator/confgenerator.go

```diff
@@ -378,6 +410,43 @@ func generateOtelExporters(exporters map[string]*MetricsExporter, pipelines map[
 	return stackdriverList, exportNameMap, nil
 }
 
+func generateOtelProcessors(processors map[string]*MetricsProcessor, pipelines map[string]*MetricsPipeline) ([]*otel.ExcludeMetrics, map[string]string, error) {
+	excludeMetricsList := []*otel.ExcludeMetrics{}
+	processorNameMap := make(map[string]string)
+	excludemetricsProcessorFactories, err := extractOtelProcessorFactories(processors)
+	if err != nil {
+		return nil, nil, err
+	}
+	for _, pID := range sortedKeys(pipelines) {
+		p := pipelines[pID]
+		for _, processorID := range p.ProcessorIDs {
+			if _, ok := processorNameMap[processorID]; ok {
+				continue
+			}
+			if p, ok := excludemetricsProcessorFactories[processorID]; ok {
+				var metricNames []string
+				for _, glob := range p.MetricPrefixes {
+					// TODO: Relax trim prefix `agent.googleapis.com/`, when it starts supporting metrics with other prefixes
```

Comment:
What's "it"? Also, a nit: missing trailing `.`. Maybe something like this?

``// TODO: Remove TrimPrefix when we support metrics with other prefixes.``